api/source/bootstrap/bootstrapUtils.js

```diff
@@ -0,0 +1,54 @@
+const path = require('path')
+const logger = require('../utils/logger')
+const config = require('../utils/config')
+const packageJson = require("../package.json")
+
+function modulePathResolver( handlersPath, route, apiDoc ) {
+    const pathKey = route.openApiRoute.substring(route.basePath.length)
+    const schema = apiDoc.paths[pathKey][route.method.toLowerCase()]
+    const controller = schema.tags[0]
+    const method = schema['operationId']
+    const modulePath = path.join(handlersPath, controller)
+    const handler = require(modulePath)
+    if (handler[method] === undefined) {
+      throw new Error(
+        `Could not find a [${method}] function in ${modulePath} when trying to route [${route.method} ${route.expressRoute}].`,
+      )
+    }
+    return handler[method]
+}
+
+function buildResponseValidationConfig() {
```

Comment:
Propose passing a boolean function argument like `willValidateResponse` and use that instead of the inline property chain from `config`